test/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 cmake_minimum_required(VERSION 3.15)
 
+set(BINARY casper_test)
```

Comment:
could you rename it to casper_cpp_sdk_tests ?

Reply:
Fixed.